OMCompiler/README.Linux.md

```diff
@@ -58,7 +85,57 @@ But first you need to install dependencies:
 - [Sundials](http://www.llnl.gov/CASC/sundials/) (optional; adds more numerical solvers to the simulation runtime)
 - libcurl (libcurl4-gnutls-dev)
 
-# Setting your environment for compiling OpenModelica
+# 2 Compile OpenModelica
+
+Now there are two options to build OpenModelica:
+
+  1. Use new CMake build.
```

Comment:
It's probably not clear to people outside the developers' inner circle what are the pro's and con's of each choice. Maybe you can quickly summarize them.

Reply:
Newer is always better? The first is always the best?
But yes, I'll recommend CMake.

Reply:
Seriously, is there any reason why we still keep the old makefile? Is it just in case CMake has issues? Or there are some cases where it is still preferable. Maybe @mahge can comment on that.

Reply:
There are some users/developers who prefer the `Makefiles` build system. Some of our CI/CD scripts such as those generating packages for Linux and Windows still uses the `Makefiles` build system. It is not that it is preferable (I guess we now have `CMake` everywhere we need it) it is just that the scripts need to updated and tested properly if we are to switch them.

My recommendation is to make sure that **we point new users to the `CMake` build system**. @AnHeuermann is doing that with these updates. 

We can also start replacing the `Makefiles` based CI/CD scripts we have one by one. It should not be a lot of work.

I even tried CMake based packaging for Windows and Debian Linux a while back and everything seems to work fine. So we can also update the packaging processes. Most of the required additions/modifications are already done when we did the packaging support for macOS. So it should be a much easier transition now.

Reply:
I think there's still an issue that OMEdit built with CMake is slow, at least for me. It takes like 20 sec to start, most of which is loading the MSL. I'm not really sure what the issue is, some bad optimization flags? But that's probably something we need to fix before starting to use CMake for packaging.

Reply:
Oh right. There is that. I am not really sure what the issue is there. Using `omc `on the `commandline` and loading Modelica is fast enough. The `omc` executable and the `OMEdit` executable both use the same exact shared library `libOpenModelicaCompiler.so` to load Modelica libraries. So perhaps the issue lies somewhere in how the OMEdit source code is compiled. There is probably some flag that is manually added to the Makefiles with such significant importance.  

The delay happens only on Linux. On Windows, IIRC, there is no such issue. I should probably investigate this properly since I got used to it and do not even notice it anymore. Which is not good.

Reply:
OK. I'm fine at directing new users to CMake. My point is that we should include a (brief) honest assessment of the pro's and con's of the two systems, otherwise most people who are not deep into the thing will have no clue.